index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+    <link rel="stylesheet" type="text/css" href="css/libs/bootstrap.min.css">
+    <link href='http://fonts.googleapis.com/css?family=Raleway|Indie+Flower' rel='stylesheet' type='text/css'> <!--Google Fonts-->
+    <link rel="stylesheet" type="text/css" href="css/libs/style.css"> <!--Icon Fonts-->
+</head>
+<body>
+    <div  class="container-fluid">
+        <div class="row">
+            <div class="col-md-1"></div>
+            <div class="col-md-10">
+
+                <header>
```

Comment:
They are semantic tags, which help to make the code easier to read. The header tag is used for the upper page content, that is, the header.

The "section" tag is used to group related content sections. In the case of my code a section  of images.